drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua

```diff
@@ -126,6 +129,19 @@ local function switch_report(driver, device, cmd)
   end
 end
 
+local function set_switch(value)
+  return function(driver, device, cmd)
+    local delay = ON_OFF_DELAY_SEC
+    local query_device = function()
+      local component = cmd and cmd.component and cmd.component or "main"
+      device:send_to_component(SwitchBinary:Get({}), component)
+    end
+
+    device:send_to_component(SwitchBinary:Set({ target_value = value, duration = 0}), cmd.component)
```

Comment:
DTH uses `Basic` for the Set, and Binary for the `Get`.

Reply:
Switch to using `Basic`